birdhouse/docker-compose.yml

```diff
@@ -266,14 +287,18 @@ services:
   mongodb:
     image: mongo:3.4.0
     container_name: mongodb
-    # Mongodb crash with permission denied errors if the command is not overriden like this
+    volumes:
+      - ${DATA_PERSIST_ROOT}/mongodb_persist:/data/db
```

Comment:
Good catch !!!  Just curious, without persist, I have not yet found something broken or lost.  Did you actually got some problems without persist?  The bigger question is, what is mongodb used for? !

Reply:
For the moment, I think most usage were for temporary data that did not matter much (if even used at all?). 
With Weaver though, deployed providers, processes and jobs execution are stored in mongo, which should be persisted.

Reply:
Mongo were used in solr, ncwms and phoenix. Phoenix stores jobs and results in it and maybe solr used to index that by the volume_from?

Reply:
As other comment. Original `mongodb` service did not even have any mapped volumes. So wherever they were writing, it was not persisted unless overridden in private config. I don't feel like any of those service should write directly into the `mongodb` managed directory either.

Reply:
I was not there when `mongodb` was added so not sure why there is no persistence but I totally agree we should add the persistence now that Weaver is using it.